packages/integration-tests/src/test-project-setup/data_storage_auth_with_triggers.ts

```diff
@@ -280,6 +287,7 @@ class DataStorageAuthWithTriggerTestProject extends TestProjectBase {
     await this.checkLambdaResponse(node16Lambda[0], expectedResponse);
     await this.checkLambdaResponse(funcWithSsm[0], 'It is working');
     await this.checkLambdaResponse(funcWithAwsSdk[0], 'It is working');
+    await this.checkLambdaResponse(funcCustomSender[0], 'Hello, World!');
```

Comment:
This is not good assertion for this feature. It just sends request directly to lambda. We should be asserting that communication between cognito and lambda works.

How to make this happen?:
1. We need to execute an operation on user pool that triggers email sending.
2. We need a function that can signal a fact that it was called by something and await that.
    1. Please take a look at solution we have for "scheduled functions". We use SQS queue to capture the fact that function was called.
        1. https://github.com/aws-amplify/amplify-backend/blob/10ef35d2fe00025fefd12603bfb2e3dce696be9a/packages/integration-tests/src/test-projects/data-storage-auth-with-triggers-ts/amplify/func-src/handler_with_aws_sqs.ts#L20-L25
        2. https://github.com/aws-amplify/amplify-backend/blob/10ef35d2fe00025fefd12603bfb2e3dce696be9a/packages/integration-tests/src/test-project-setup/data_storage_auth_with_triggers.ts#L614
        3. And fail with timeout (if sqs message doesn't arrive),